ext/mysqli/mysqli_warning.c

```diff
@@ -142,13 +142,13 @@ MYSQLI_WARNING * php_get_warnings(MYSQLND_CONN_DATA * mysql)
 
 		/* 1. Here comes the error no */
 		entry = zend_hash_get_current_data(Z_ARRVAL(row));
-		errno = zval_get_long(entry);
+		mysqli_errno = zval_get_long(entry);
 		zend_hash_move_forward(Z_ARRVAL(row));
 
 		/* 2. Here comes the reason */
 		entry = zend_hash_get_current_data(Z_ARRVAL(row));
 
-		w = php_new_warning(entry, errno);
+		w = php_new_warning(entry, mysqli_errno);
```

Comment:
The errno changes (in ldap and mysqli) also look good, please commit separately.